articles/azure-functions/durable-functions-external-events.md

```diff
@@ -22,7 +22,7 @@ Orchestrator functions have the ability to wait and listen for external events.
 
 ## Wait for events
 
-The [WaitForExternalEvent](https://azure.github.io/azure-functions-durable-extension/api/Microsoft.Azure.WebJobs.DurableOrchestrationContext.html#Microsoft_Azure_WebJobs_DurableOrchestrationContext_WaitForExternalEvent_) method allows an orchestrator function to asynchronously wait and listen for an external event. The caller declares the *name* of the event and the *shape of the data* it expects to receive.
+The [WaitForExternalEvent](https://azure.github.io/azure-functions-durable-extension/api/Microsoft.Azure.WebJobs.DurableOrchestrationContext.html#Microsoft_Azure_WebJobs_DurableOrchestrationContext_WaitForExternalEvent_) method allows an orchestrator function to asynchronously wait and listen for an external event. The listening orchestrator function declares the *name* of the event and the *shape of the data* it expects to receive.
```

Comment:
It's the listener that determines the shape of the data it expects to recive, not the caller.

Reply:
By "caller" I meant the code which calls `WaitForExternalEvent`, but your suggestion removes ambiguity.